src/Breadcrumbs/DatabaseQueryExecuted.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+
+namespace Honeybadger\HoneybadgerLaravel\Breadcrumbs;
+
+use Honeybadger\HoneybadgerLaravel\Concerns\HandlesEvents;
+use Honeybadger\HoneybadgerLaravel\Facades\Honeybadger;
+use Illuminate\Database\Events\QueryExecuted;
+
+class DatabaseQueryExecuted
+{
+    use HandlesEvents;
+
+    public $handles = QueryExecuted::class;
+
+    public function handleEvent(QueryExecuted $event)
+    {
+        $metadata = [
+            'connectionName' => $event->connectionName,
+            'sql' => $event->sql,
```

Comment:
I think the Ruby gem may do some extra sanitizing of SQL queries (but I could be wrong). Could you go check on that just to be safe? We need to be sure that we won't be leaking any sensitive data.

/cc @rabidpraxis (in case you have thoughts)

Reply:
That's correct, we obfuscate all queries, which removes most of the details (https://github.com/honeybadger-io/honeybadger-ruby/blob/master/lib/honeybadger/util/sql.rb). 

It is often not the most useful, but we aired on the side of caution, especially since customers could have this data added to HB with just an upgrade.

Reply:
This isn't the bound query, though. This is the query with `?` s for all parameters. There's a `binding` key on the event that holds the actual parameters, such I didn't include.

Reply:
It is the same with Rails, but we also go a step further to remove all quoted strings & numbers, essentially leaving the SQL structure.

I'm not saying it's the right thing to do. It is conservative and often strips out clues for debugging. I believe we made that decision at the time to be super safe in-case some quoted params slipped through.

Reply:
Okay, that makes sense. I'll look into doing that.

Reply:
Done, but I feel a bit weird about turning a  safe Laravel query like `DB::table('users')->select('email')->take(2)->get()` to `select ? from ? limit ?`.😥 But I guess if it serves the customer's utility...